i18n/locales/italian/translations.json

```diff
@@ -32,23 +32,35 @@
     "go-to-front-page": "Vai alla Home"
   },
   "social-row": {
-    "author-has-twitter": "Se hai letto fino a qui, twitta al traduttore per ringraziarlo. {{ <0> }}Twitta un ringraziamento{{ </0> }}",
-    "author-no-twitter": "Se questo articolo ti è stato utile, {{ <0> }}twittalo{{ </0> }}.",
-    "default-tweet": "Grazie {{ twitter }} per avere tradotto questo utile articolo."
+    "cta": {
+      "tweet-a-thanks": "Se hai letto fino a qui, twitta al traduttore per ringraziarlo. {{ <0> }}Twitta un ringraziamento{{ </0> }}",
```

Comment:
```suggestion
      "tweet-a-thanks": "Se hai letto fino a qui, {{ <0> }}scrivi un tweet di ringraziamento!"{{ </0> }}",
```

Reply:
I'll do this one manually if that's alright.